ovssubnet/controller/kube/kube.go

```diff
@@ -22,11 +22,11 @@ func NewFlowController() *FlowController {
 	return &FlowController{}
 }
 
-func (c *FlowController) Setup(localSubnet, containerNetwork, servicesNetwork string) error {
+func (c *FlowController) Setup(localSubnet, containerNetwork, servicesNetwork string, mtu uint) error {
 	_, ipnet, err := net.ParseCIDR(localSubnet)
 	subnetMaskLength, _ := ipnet.Mask.Size()
 	gateway := netutils.GenerateDefaultGateway(ipnet).String()
-	out, err := exec.Command("openshift-sdn-kube-subnet-setup.sh", gateway, ipnet.String(), containerNetwork, strconv.Itoa(subnetMaskLength), gateway).CombinedOutput()
+	out, err := exec.Command("openshift-sdn-kube-subnet-setup.sh", gateway, ipnet.String(), containerNetwork, strconv.Itoa(subnetMaskLength), gateway, fmt.Sprint(mtu)).CombinedOutput()
```

Comment:
awkward that we're using both strconv.Itoa() and fmt.Sprint() to convert integers to strings in the same line...